### PR TITLE
Feature/update drone pipeline

### PR DESCRIPTION
**Pull request by JoeChapman**
- add badges to readme
- add image_to_quay pipeline
- run drone sign

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -5,22 +5,36 @@ pipeline:
       - npm --loglevel warn install -g npm@3
       - npm --loglevel warn install
       - npm run test:ci
+    when:
+      event: [push, pull_request]
```
Comment (joefitter) [inline .drone.yml]: aaah! that's how you do it!

Comment (JoeChapman) [inline .drone.yml]: yep

File: .drone.yml [hunk 2]
```diff
@@ -30,16 +44,49 @@ pipeline:
...
+    when:
+      branch: master
+      event: push
+
+  image_to_quay:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker login -u="ukhomeofficedigital+ukvi_complaints" -p=${DOCKER_PASSWORD} quay.io
```
Comment (joefitter) [inline .drone.yml]: we are passing in the docker password - would it make sense to also pass in the app name? That way the same file could be shared across applications

File: .drone.yml [hunk 3]
```diff
@@ -30,16 +44,49 @@ pipeline:
...
+      - docker tag app quay.io/ukhomeofficedigital/ukvi-complaints:${DRONE_COMMIT_SHA}
```
Comment (joefitter) [inline .drone.yml]: same here

File: .drone.yml [hunk 4]
```diff
@@ -30,16 +44,49 @@ pipeline:
...
+      - docker push quay.io/ukhomeofficedigital/ukvi-complaints:${DRONE_COMMIT_SHA}
```
Comment (joefitter) [inline .drone.yml]: and here

Comment (joefitter) [inline .drone.yml]: maybe we could have the quay address as an env var?

File: .drone.yml (hunk 2 above)
Comment (JoeChapman) [inline .drone.yml]: what do you mean by the same file? Which file?

Comment (joefitter) [inline .drone.yml]: .drone.yml - currently we have a couple of ukvi-specific bits in it - just wondering if we could abstract them out as env vars and then the same drone config could be used across applications?

Comment (JoeChapman) [inline .drone.yml]: yes we will be able to, but that is out of scope of this project. Abstraction and automation are on the cards

Comment (JoeChapman) [inline .drone.yml]: In order to do what you suggest using env vars, I think we'd need to set drone secrets, which is probably something devops would want to take charge of. I'll look into it

File: .drone.yml (hunk 4 above)
Comment (JoeChapman) [inline .drone.yml]: I have just spoken to the devops who say we can manage this so it's really simple. We just abstract this into secrets, i.e. in this example we'd need `${DOCKER_USER}` and `${QUAY_REPO}`

Comment (JoeChapman) [inline .drone.yml]: `drone secret add <repo_name> --image=docker:1.11 DOCKER_USER ukhomeofficedigital+ukvi_complaints`
